tests/Polkadot/SignerTests.cpp

```diff
@@ -19,7 +19,7 @@
 
 namespace TW::Polkadot {
     auto privateKey = PrivateKey(parse_hex("0xabf8e5bdbe30c65656c0a3cbd181ff8a56294a69dfedd27982aace4a76909115"));
-    auto toPublicKey = PublicKey(parse_hex("0x8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48"), TWPublicKeyTypeED25519);
+    auto toPublicKey = PublicKey(parse_hex("0xbeff0e5d6f6e6e6d573d3044f3e2bfb353400375dc281da3337468d4aa527908"), TWPublicKeyTypeED25519);
```

Comment:
hmm, I got this public from polkadot.js lib, why this one was not valid?

Reply:
Probably this value is not a valid public key (no private key exists that would generate it), but it is a bit worrisome why this value was used in polkadot test.

Reply:
Polkadot tests reverted.